rest/utilities_testing_test.go

```diff
@@ -99,3 +103,22 @@ func TestAttachmentRoundTrip(t *testing.T) {
 	assert.Equal(t, []byte{}, attachments["baz"].Data) // data field is explicitly ignored
 
 }
+
+func TestCECheck(t *testing.T) {
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("Only works with CBS")
+	}
+	if base.TestsUseServerCE() {
+		rt := NewRestTester(t, nil)
+		defer rt.Close()
+		form := url.Values{}
+		form.Add("password", "password")
+		form.Add("roles", "[mobile_sync_Gateway]")
+		eps, _, err := rt.ServerContext().ObtainManagementEndpointsAndHTTPClient()
+		require.NoError(t, err)
+
+		req, err := http.NewRequest("PUT", fmt.Sprintf("%s/settings/rbac/users/local/%s", eps[0], "username"), strings.NewReader(form.Encode()))
+		require.Error(t, err)
+		require.Equal(t, req, http.StatusBadRequest)
+	}
```

Comment:
```suggestion
	if base.TestsUseServerCE() {
		t.Skip("test only runs with CE server")
	}
	rt := NewRestTester(t, nil)
	defer rt.Close()
	form := url.Values{}
	form.Add("password", "password")
	form.Add("roles", "[mobile_sync_Gateway]")
	eps, _, err := rt.ServerContext().ObtainManagementEndpointsAndHTTPClient()
	require.NoError(t, err)

	req, err := http.NewRequest("PUT", fmt.Sprintf("%s/settings/rbac/users/local/%s", eps[0], "username"), strings.NewReader(form.Encode()))
	require.Error(t, err)
	require.Equal(t, req, http.StatusBadRequest)
	
```

in idiomatic go, prefer the early return for readability.